js-exercises/sum-even-args/sumEvenArgs.js

```diff
@@ -0,0 +1,14 @@
+const sumEvenArgs = (...args) => {
+  if(args.length === 0) return 0;
+
+  let sum = 0;
+  for(let arg of args){
+    if(isNaN(arg)) continue;
+
+    if(arg % 2 === 0) sum += arg;
```

Comment:
I think you should check these two conditions in one if condition, otherwise it makes workflow complex.